google-cloud-bigquery/clirr-ignored-differences.xml

```diff
@@ -4,32 +4,12 @@
   <!-- TODO: REMOVE AFTER RELEASE -->
   <difference>
     <differenceType>7013</differenceType>
-    <className>com/google/cloud/bigquery/MaterializedViewDefinition</className>
-    <method>com.google.cloud.bigquery.Clustering getClustering()</method>
```

Comment:
Why remove the block for materialized view?

Reply:
This is not necessary anymore since it has already gone into the previous release and formed a new baseline.